homeassistant/components/camera/__init__.py

```diff
@@ -411,7 +426,17 @@ class CameraMjpegStream(CameraView):
     url = '/api/camera_proxy_stream/{entity_id}'
     name = 'api:camera:stream'
 
-    @asyncio.coroutine
-    def handle(self, request, camera):
-        """Serve camera image."""
-        yield from camera.handle_async_mjpeg_stream(request)
+    async def handle(self, request, camera):
+        """Serve camera stream, possibly with interval."""
+        if request.query.get('interval'):
+            # Compose camera stream from stills
+            interval = float(request.query.get('interval'))
+            if interval < MIN_STREAM_INTERVAL:
+                return web.Response(status=400,
+                                    reason="Interval must be a number \
```

Comment:
Reason should be the status code. I think that you're looking for something like `self.json_message`